core/workspace_svg.js

```diff
@@ -1110,6 +1110,17 @@ Blockly.WorkspaceSvg.prototype.isDeleteArea = function(e) {
   return Blockly.DELETE_AREA_NONE;
 };
 
+/**
+ * Is the mouse event outside the blocks UI, to the right of the workspace?
+ * @param {!Event} e Mouse move event.
+ * @return {boolean} True if event is outside the blocks UI.
+ */
+Blockly.WorkspaceSvg.prototype.isOutside = function(e) {
+  var mousePoint = Blockly.utils.mouseToSvg(e, this.getParentSvg(),
+      this.getInverseScreenCTM());
```

Comment:
Why are you converting to SVG here?  You started with a mouse point, and your concern is about window coordinates, not workspace coordinates.  This may be unnecessarily expensive for what you're actually trying to do.

Reply:
I copied this from somewhere since it seemed to give me the right number. What is the difference between the SVG coordinates and mouse coordinates?